### ISSUE TITLE
Contact - Client Photo field Delete

### ISSUE THREAD

**Issue by fifedog** [fifedog/hartOfFolsom#30]
We can remove the production Client field photo, and hide the ProfileIMGURL from the page layout.

**Comment by fifedog** [comment]
@rsharley - would you do this when you're pushing your stuff to production?

**Comment by fifedog** [comment]
@rsharley - I think this is done right?

**Comment by rsharley** [comment]
I think so but let me check before I close it out. 
Cheers,
Rich


Sent from AT&T Yahoo Mail for iPhone


On Monday, June 22, 2020, 5:37 PM, Peter Fife <notifications@github.com> wrote:




@rsharley - I think this is done right?

**Comment by rsharley** [comment]
Peter,
The ProfileIMGURL field was removed from the layout. 
When I removed that field from the layout I could not find the production Client field photo you referenced in the github issue. Am I missing something?
Rich


Sent from AT&T Yahoo Mail for iPhone


On Monday, June 22, 2020, 6:42 PM, Rich Smith <rich.smith@pacbell.net> wrote:

I think so but let me check before I close it out. 
Cheers,
Rich


Sent from AT&T Yahoo Mail for iPhone


On Monday, June 22, 2020, 5:37 PM, Peter Fife <notifications@github.com> wrote:




@rsharley - I think this is done right?
